core/azure-core-amqp/pom.xml

```diff
@@ -7,13 +7,13 @@
   <parent>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-parent</artifactId>
-    <version>1.0.0-preview.1</version>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
For now this should be 1.1.0 to match what is in the repo, at least until there are changes and it's version is bumped.